net.biniou.LeBiniou.yml

```diff
@@ -0,0 +1,86 @@
+app-id: net.biniou.LeBiniou
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: lebiniou
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --filesystem=xdg-run/pipewire-0
+  # used to send anonymous usage statistics
+  - --share=network
+  # users can put configuration and/or data files anywhere
+  - --filesystem=home
+  # needed to access the webcam
+  - --device=all
+modules:
+  - name: microhttpd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.wayne.edu/gnu/libmicrohttpd/libmicrohttpd-0.9.71.tar.gz
+        sha256: e8f445e85faf727b89e9f9590daea4473ae00ead38b237cf1eda55172b89b182
+  - name: orcania
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/babelouest/orcania/archive/v2.1.1.tar.gz
+        sha256: 960074f6aa198e9db5440347d20cff8be3b82ee1c15ac7b23c0ac54a301a85cd
+  - name: yder
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/babelouest/yder/archive/v1.4.12.tar.gz
+        sha256: 59bed6df43c29bf1a0f85274369f7a8bc7dfe5e94be826139e8aa05eda3ba2e6
+  - name: jansson
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/akheron/jansson/archive/v2.13.1.tar.gz
+        sha256: f22901582138e3203959c9257cf83eba9929ac41d7be4a42557213a22ebcc7a0
+  - name: ulfius
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/babelouest/ulfius/archive/v2.7.1.tar.gz
+        sha256: d5dfb90ac16fe9d8ce70fe6b23e43102d5208d4f0174196d3ef183d950d3a57b
+  - name: imagemagick
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick6/archive/6.9.11-51.tar.gz
+        sha256: 06c4363486e1053e69bb3fa211374bdaed4baa678ca8ccd1f38731f0da2d3497
+  - name: jackaudio
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/jackaudio/jack2/archive/v1.9.16.tar.gz
+        sha256: e176d04de94dcaa3f9d32ca1825091e1b938783a78c84e7466abd06af7637d37
+    build-commands:
+      - ./waf configure --prefix=/app
+      - ./waf
+      - ./waf install
+  - name: lebiniou-data
+    buildsystem: simple
+    sources:
+       - type: git
+         url: https://gitlab.com/lebiniou/lebiniou-data.git
+         tag: version-3.50
+         commit: f4b03c90b838c4b84ab2cd56348dedf724e653b2
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
```

Comment:
This should be $FLATPAK_BUILDER_N_JOBS like so: https://github.com/linuxhw/hw-probe/blob/07ff9f20b5ebc23db596691037d04e0eef460f3b/flatpak/org.linux_hardware.hw-probe.yaml#L81